docker_start.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+if [ -z "${TARGET_URL}" ]; then
```

Comment:
While I agree we should handle env vars for all of these settings, I think we should handle these at the app leveland simply take whatever the `$@` entrypoint is, defaulting to `locust`.  This allows the user to modify their entry as necessary and simply pass the envvars to be used.